packages/suite-native-settings/src/components/DeviceSettings.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { ListItem } from '@suite-native/atoms';
+
+import { SettingsSection } from './SettingsSection';
+
+import { SettingItem } from '../types';
+
+const deviceSettingItems: SettingItem[] = [
+    { title: 'Customization', description: 'Name, Homescreen', iconName: 'palette' },
+    { title: 'Security', description: 'PIN Active, Passphrase enabled', iconName: 'lock' },
+    { title: 'Danger Area', description: 'Factory reset, Custom firmware', iconName: 'lock' },
+];
+
+export const DeviceSettings = () => (
+    <SettingsSection title="Device">
+        {deviceSettingItems.map(item => (
+            <ListItem
```

Comment:
Please create component called SettingsSectionItem, it will be mostly wrapped ListItem, but it will simplify things and it will be sematically better:

```tsx
// please move SettingsItem type from types.ts here

const SettingsSectionItem = (props: ListItemProps & SettingsItem) => <ListItem hasRightArrow {...props} />
```

Reply:
[e035a1b](https://github.com/trezor/trezor-suite/pull/5617/commits/e035a1b7e5eabc6a3f1ee38466cc48311713970f)